.github/workflows/lint.yml

```diff
@@ -91,3 +91,12 @@ jobs:
     - name: lint remain
       run: |
         make lint-remain
+
+    - name: shellcheck
+      uses: reviewdog/action-shellcheck@v1
+      with:
+        github_token: ${{ secrets.github_token }}
+        reporter: github-pr-review
+        path: "."
+        pattern: "*.sh"
+        exclude: "./.git/*"
```

Comment:
We don't have shell script under this directory.

Reply:
we should set `exclude` to the directories which we do want to check. 
Like we do not need to check `.git` directory here.
ref: https://github.com/reviewdog/action-shellcheck?tab=readme-ov-file#exclude

Reply:
We already use `*.sh` to limit the file we will check. Unless there is `.git/xx.sh`, we don't need to add another filter.

Reply:
agree, done